### PR TITLE
Meta: add some missing meeting minutes

### PR DESCRIPTION
**Pull request by miketaylr**
Minutes for August, October, November, and December meetings. No meeting happened in September, but I don't remember why.

### CODE REVIEW

**Comment by myakura** [comment]
> No meeting happened in September, but I don't remember why.

TPAC, perhaps?

**Comment by miketaylr** [comment]
> > No meeting happened in September, but I don't remember why.
> 
> TPAC, perhaps?

Aha! That is probably it. :)

File: uastring/minutes/2022-12-16-minutes.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
+# [Minutes] UA String Reduction - 2022-11-18
```
Comment (karlcow) [inline uastring/minutes/2022-12-16-minutes.md:1]: Probably change the date
```suggestion
# [Minutes] UA String Reduction - 2022-12-16
```

**Review by karlcow — CHANGES_REQUESTED** [review]
Date to change for December,
And the index needs to be updated too for listing the recent minutes.

**Review by karlcow — APPROVED** [review]
Thanks. Still the index missing, but we can probably add it later. 🍾

**Comment by miketaylr** [comment]
> Thanks. Still the index missing, but we can probably add it later. 🍾

whoops - just a bit slow on vacation.

**Pull request merged**